src/reducers/books.js

```diff
@@ -1,12 +1,13 @@
+/* eslint-disable object-curly-newline */
```

Comment:
- Disabling eslint will make you miss out on coding practices that Microverse seeks to let you know.